Documentation/installation/k8s-install-openshift-okd.rst

```diff
@@ -10,255 +10,12 @@
-   INFO Consuming Common Manifests from target directory
-   INFO Credentials loaded from the "default" profile in file "/home/twp/.aws/credentials"
-   INFO Creating infrastructure resources...
-   INFO Waiting up to 20m0s for the Kubernetes API at https://api.cluster-name.ilya-openshift-test-1.cilium.rocks:6443...
-   INFO API v1.20.0-1058+7d0a2b269a2741-dirty up
-   INFO Waiting up to 30m0s for bootstrapping to complete...
-   INFO Destroying the bootstrap resources...
-   INFO Waiting up to 40m0s for the cluster at https://api.cluster-name.ilya-openshift-test-1.cilium.rocks:6443 to initialize...
-   INFO Waiting up to 10m0s for the openshift-console route to be created...
-   INFO Install complete!
-   INFO To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/twp/okd/cluster-name/auth/kubeconfig'
-   INFO Access the OpenShift web-console here: https://console-openshift-console.apps.cluster-name.ilya-openshift-test-1.cilium.rocks
-   INFO Login to the console with user: "kubeadmin", and password: "<REDACTED>"
-   INFO Time elapsed: 32m9s
-
-Accessing the cluster
----------------------
-
-To access the cluster you will need to use ``kubeconfig`` file from the ``${CLUSTER_NAME}/auth`` directory:
-
-.. code-block:: shell-session
-
-   export KUBECONFIG="${CLUSTER_NAME}/auth/kubeconfig"
-
-Prepare cluster for Cilium connectivity test
---------------------------------------------
-
-In order for Cilium connectivity test pods to run on OpenShift, a simple custom ``SecurityContextConstraints``
-object is required. It will to allow ``hostPort``/``hostNetwork`` that some of the connectivity test pods rely on,
-it sets only ``allowHostPorts`` and ``allowHostNetwork`` without any other privileges.
-
-.. code-block:: shell-session
-
-   kubectl apply -f - <<EOF
-   apiVersion: security.openshift.io/v1
-   kind: SecurityContextConstraints
-   metadata:
-     name: cilium-test
-   allowHostPorts: true
-   allowHostNetwork: true
-   users:
-     - system:serviceaccount:cilium-test:default
-   priority: null
-   readOnlyRootFilesystem: false
-   runAsUser:
-     type: MustRunAsRange
-   seLinuxContext:
-     type: MustRunAs
-   volumes: null
-   allowHostDirVolumePlugin: false
-   allowHostIPC: false
-   allowHostPID: false
-   allowPrivilegeEscalation: false
-   allowPrivilegedContainer: false
-   allowedCapabilities: null
-   defaultAddCapabilities: null
-   requiredDropCapabilities: null
-   groups: null
-   EOF
-
-Deploy the connectivity test
-----------------------------
-
-.. include:: kubectl-connectivity-test.rst
-
-Cleanup after connectivity test
--------------------------------
-
-Remove the ``SecurityContextConstraints``:
-
-.. code-block:: shell-session
-
-   kubectl delete scc cilium-test
-
-Delete the cluster
-------------------
-
-.. code-block:: shell-session
-
-   openshift-install destroy cluster --dir="${CLUSTER_NAME}"
+Cilium OLM images for installation on OpenShift can be found on the Red Hat
```

Comment:
This sentence seems to just repeat the previous paragraph, can we delete it?

Reply:
Ah I accidentally left this in, nice catch.